manage_rbac.py

```diff
@@ -20,15 +20,15 @@ class RBACManagerException(Exception):
 
 class RBACManager(object):
 
-    def __init__(self, *args, **kwargs):
-        pass
+    def __init__(self, namespace, *args, **kwargs):
```

Comment:
I'd like make this not required, maybe use kwargs to pass in the namespace and have it default to the namespace the controller is deployed in.

Reply:
Since we have to know what namespace to use when looking for rbacdefs, this will have to stay for now.  Per other discussions, this will go away when the CRD is clustered instead of namespaced.

Reply:
I disagree. It does not need to stay. I'm not convinced the resource should be cluster-wide, and we do not need to make it required since we can default to the namespace the controller pod is running in.

Reply:
This does default to the namespace of the pod. This is how we know what that namespace is (passed in from environment or cli)

Reply:
I was assuming there would be a way to get the current namespace from the python client but there does not appear to be. Carry on.